swiftgen-cli/ArgumentsUtils.swift

```diff
@@ -47,12 +47,20 @@ enum OutputDestination: ArgumentConvertible {
     }
   }
 
-  func write(content: String) {
+  func writeIfChanged(content: String) {
+    write(content, onlyIfChanged: true)
+  }
+  
+  func write(content: String, onlyIfChanged: Bool = false) {
     switch self {
     case .Console:
       print(content)
     case .File(let path):
       do {
+        if try path.isContentEqualTo(content) && onlyIfChanged {
```

Comment:
Actually, this does not seem to be possible. When the order is inverted `onlyIfChanged && try path.isContentEqualTo(content)` there is a build error  `'try' cannot appear to the right of a non-assignment operator'. 

Is there a way of achieving this behaviour other than to use nested `if` statements?